{
    "description": "联系人：检查是否能正常进入联系人应用",
    "environment": [
        {
            "type": "device",
            "label": "phone"
        }
    ],
    "driver": {
        "type": "DeviceTest",
        "py_file": ["Contacts.py"]
    }
}